components/output/slow_pwm.rst

```diff
@@ -0,0 +1,41 @@
+Slow PWM Output
+===============
+
+.. seo::
+    :description: Instructions for setting up slow pwm outputs for GPIO pins.
+    :image: pwm.png
+
+Similar to PWM, The Slow PWM Output platform allows you to control GPIO pins by
+pulsing them on/off over a longer time period. It could be used to control a
+heating element through a relay where a fast PWM update cycle would not be appropriate.
+
```

Comment:
```rst
.. note::

    This is for **slow** PWM output. For fast-switching PWM outputs (for example for lights), see these outputs:

    - ESP32: :doc:`ledc`
    - ESP8266: :doc:`esp8266_pwm`
```

Quite a few people (unfortunately) don't read the top paragraph. And from the index page it's rather simple to get here (has the same symbol as other PWM pages, which is ok) - so let's add a separate note here just to be sure.